sliceop/f64s/example_f64s_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright ©2020 The go-hep Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package f64s_test
+
+import (
+	"fmt"
+)
```

Comment:
you're also missing an import of `"go-hep.org/x/hep/sliceop/f64s"`

```suggestion

	"go-hep.org/x/hep/sliceop/f64s"
)
```